presto-blackhole/src/main/java/io/prestosql/plugin/blackhole/BlackHoleMetadata.java

```diff
@@ -253,7 +256,7 @@ public ConnectorTableProperties getTableProperties(ConnectorSession session, Con
 
     private void checkSchemaExists(String schemaName)
     {
-        if (!schemas.contains(schemaName)) {
+        if (!schemas.contains(createNonDelimitedName(schemaName))) {
```

Comment:
Make `checkSchemaExists` take `Name schemaName` instead (since it's called by `beginCreateTable`, which passes `tableMetadata.getTable().getSchemaName()` as the argument)